lib/git_worktree.rb

```diff
@@ -37,17 +37,11 @@ def initialize(options = {})
     @remote_name = 'origin'
     @base_name   = File.basename(@path)
 
-    # The libssh2 library must already be installed at gem installation time
-    # for the 'ssh' feature to be available.
+    # The libssh2 library must already be installed and the `--with-ssh` build option
+    # passed at gem installation time for the 'ssh' feature to be available.
     #
-    # For Fedora/Centos, the presence of libssh2 seems to be enough to get it
-    # to build properly.
-    #
-    # For OSX:
-    #
-    #     $ brew install libssh2
     #     $ gem uninstall rugged  # if required
-    #     $ export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/openssl/lib/pkgconfig"
+    #     $ bundle config set --local build.rugged --with-ssh
     #     $ bin/bundle
```

Comment:
Not sure if this which is more specific vs just telling people to `gem uninstall && bin/update`

Reply:
Yeah I like this more specific documentation.

Reply:
Why `bin/bundle`?  Should we change that to `bundle` or `bundle install`?  I think we want to always use newer bundlers, so if you have newer ones use that and not the last one specified in the `Gemfile.lock`.  If bundler releases a bad version, we need to detect it and disable that specific version until we can resolve the problem.

Reply:
Because that's what was already here and wasn't related to my changes :shrug: not a great reason but didn't think it needed to change.
I can change to `bundle install` but should we delete `bin/bundle` if it shouldn't be used?  (disclosure I never use it but not sure why it was there then)

Reply:
Actually, bin/update would be fine here too.  I like the new changes but we should probably pick `bundle install` (for similar behavior as before but with the latest bundler on the system) or `bundle update` (if we think this is better)